### ISSUE TITLE
My Gaia

### ISSUE THREAD

**Issue by FoxXT** [lordsolrac/GaiaOnlineDark#1]
The Gaia Overview Could Be removed with CSS as all the items are underneath **My Gaia.** and because it breaks the theme you're trying to make :)

**Comment by lordsolrac** [comment]
Mind providing a screenshot? What extra extenisons were you using and the browser as well.
Did you try the latest version (0.4a as of this writing)

**Comment by FoxXT** [comment]
http://pad1.whstatic.com/images/thumb/1/14/Get-Lots-of-Gold-on-GaiaOnline-Step-1-Version-2.jpg/aid1184780-728px-Get-Lots-of-Gold-on-GaiaOnline-Step-1-Version-2.jpg

Ignore the arrow, but the Gaia overview is the bar containing inventory, avatar, car, etc. **Not the site navigation bar above.**

**Comment by lordsolrac** [comment]
If you can provide your view of it I could find it easier. Are you referring to the menus on the MyGaia section?

**Comment by FoxXT** [comment]
If you go to my Gaia (the page, not the menu) you'll see a bar above the friends online and ad. Its to the left of the profile link. It has all the icons.

**Comment by lordsolrac** [comment]
You're reffering to the icons of MyGaia's navigation (MyGaia throughout Settings, the icons)?
I believe that's been long fixed. 
http://imgur.com/xK3pPnOl.png

**Comment by FoxXT** [comment]
Yes. The White makes the theme feel out of place. I'd recommend  inverting the colors of it, and or removing it.

**Comment by lordsolrac** [comment]
I honestly thought you meant they were missing (Whoops~) a LOT of Gaia's sprite/image works require fixing,  they are raster images, I'll have them transparent once I'm done fixing all the css-exclusive bugs
